### DISCUSSION TITLE
Is there a dedicated filetype for floating window opened by the plugin?

### DISCUSSION THREAD

**Discussion by skoch13** [luckasRanarison/nvim-devdocs#36] [Q&A] [answered]
I want to turn off `mini-indentscope` for floating windows opened by nvim-devdocs. To do so, I need to provide a filetype, as shown here: [https://www.lazyvim.org/plugins/ui#miniindentscope](https://www.lazyvim.org/plugins/ui#miniindentscope).

![image](https://github.com/luckasRanarison/nvim-devdocs/assets/29177689/6506d2ed-709f-423b-9f97-61d69029b167)

Could you please guide me on how to proceed?

**Comment by luckasRanarison** [comment, marked as answer]
I think it's possible with the new option from [this PR](https://github.com/luckasRanarison/nvim-devdocs/pull/34), you could add a callback with `after_open` and set a custom filetype like this:
```lua
{
-- other options here
after_open = function(bufnr)
  vim.bo[bufnr].ft = "my_custom_ft"
end
}
```
Then you could just add this custom filetype to your ignore list.

**Reply by skoch13** [reply]
@luckasRanarison Thank you!
